docs/configuration.md

```diff
@@ -0,0 +1,259 @@
+MAGENTO_BASE_URL=http://magento2.vagrant251
+```
+
+<div class="bs-callout bs-callout-info" markdown="1">
+If the `MAGENTO_BASE_URL` contains a subdirectory (like `http://magento.test/magento2ce`), specify [`MAGENTO_CLI_COMMAND_PATH`][].
+</div>
+
+### MAGENTO_BACKEND_NAME
+
+The path to the Magento Admin page.
+
+Example:
+
+```conf
+MAGENTO_BACKEND_NAME=admin_12346
+```
+
+### MAGENTO_ADMIN_USERNAME
+
+The username that tests can use to access the Magento Admin page
+
+Example:
+
+```conf
+MAGENTO_ADMIN_USERNAME=admin
+```
+
+### MAGENTO_ADMIN_PASSWORD
+
+The password that tests will use to log in to the Magento Admin page.
+
+Example:
+
+```conf
+MAGENTO_ADMIN_PASSWORD=1234reTyt%$7
+```
+
+## Advanced configuration
+
+Depending on the environment you use, you may need to configure the MFTF more precisely by setting more configuration parameters then for basic configuration.
+This section describes available configuration parameters and their default values (where applicable).
+
+### DEFAULT_TIMEZONE
+
+Sets a default value for the `timezone` attribute of a [`generateDate` action][generateDate].
+This value is applied when a test step does not specify a time zone.
+For the complete list of available time zones, refer to [List of Supported Timezones][timezones].
+
+Default: `America/Los_Angeles`.
+
+Example:
+
+```conf
+DEFAULT_TIMEZONE=UTC
+```
+
+### SELENIUM
+
+The `SELENIUM_*` values form the URL of a custom Selenium server for running testing.
+
+Default Selenium URL: `http://127.0.0.1:4444/wd/hub`
+
+And the default configuration:
+
+```conf
+SELENIUM_HOST=127.0.0.1
+SELENIUM_PORT=4444
+SELENIUM_PROTOCOL=http
+SELENIUM_PATH=/wd/hub
+```
+
+<div class="bs-callout bs-callout-warning" markdown="1">
+`SELENIUM_*` values are required if you are running Selenium on an external system.
+If you change the configuration of the external Selenium server, you must update these values.
+</div>
+
+#### SELENIUM_HOST
+
+Override the default Selenium server host.
+
+Example:
+
+```conf
+SELENIUM_HOST=user:pass@ondemand.saucelabs.com
+```
+
+#### SELENIUM_PORT
+
+Override the default Selenium server port.
+
+Example:
+
+```conf
+SELENIUM_PORT=443
+```
+
+#### SELENIUM_PROTOCOL
+
+Override the default Selenium server protocol.
+
+Example:
+
+```conf
+SELENIUM_PROTOCOL=https
+```
+
+#### SELENIUM_PATH
+
+Override the default Selenium server path.
+
+Example:
+
+```conf
+SELENIUM_PATH=/wd/hub
+```
+
+### MAGENTO_RESTAPI
+
+These `MAGENTO_RESTAPI_*` values are optional and can be used in cases when your Magento instance has a different API path than the one in `MAGENTO_BASE_URL`.
+
+```conf
+MAGENTO_RESTAPI_SERVER_HOST
+MAGENTO_RESTAPI_SERVER_PORT
+```
+
+#### MAGENTO_RESTAPI_SERVER_HOST
+
+The protocol and the host of the REST API server path.
+
+Example:
+
+```conf
+MAGENTO_RESTAPI_SERVER_HOST=http://localhost
+```
+
+#### MAGENTO_RESTAPI_SERVER_PORT
+
+The port part of the API path.
+
+Example:
+
+```conf
+MAGENTO_RESTAPI_SERVER_PORT=5000
+```
+
+### *_BP
```

Comment:
```suggestion
### \*_BP
```
escape * as a special symbol in markdown

Reply:
Nice catch.